Add 3 entries to the `/knowledge` center (#1917)
vercel/docs: pages/knowledge/can-i-add-nameservers-to-subdomains.mdx

import Layout from '~/components/layout/knowledge'
import Note from '~/components/text/note'

export const meta = {

>   title: 'Can I add NS records to subdomains on Vercel?',

Reviewer: ```suggestion
  title: 'Can I Add Nameserver Records to Subdomains on Vercel?',
```

  description:
    'Information about subzone delegation when using Nameservers managed by Vercel.',
  editUrl: 'pages/knowledge/can-i-add-nameservers-to-subdomains.mdx',
  url: '/knowledge/can-i-add-nameservers-to-subdomains',

>   published: '2020-06-15T13:46:54.155Z',

Reviewer: ```suggestion
  published: '2020-06-17T13:46:54.155Z',
```

  lastEdited: '2020-06-15T13:46:54.155Z'
}

The Vercel platform does not support Nameservers on subdomains (subzone delegation). We recommend customers with this specific need to use another DNS provider.

> You can still use [A records](/knowledge/a-record-and-caa-with-vercel) with your domain to host your site with Vercel while the Nameservers are managed by a 3rd party company.

Reviewer: ```suggestion

## Managing DNS Externally

If you choose to manage your DNS outside of Vercel to make use of subzone delegation you can verify your domain for use with Vercel by adding an A record.

You can find more information on adding an A record to your external domain while using it to host a Vercel project [here](/knowledge/a-record-and-caa-with-vercel).
```
